disnake/client.py

```diff
@@ -352,23 +360,39 @@ def __init__(
         *,
         asyncio_debug: bool = False,
         loop: Optional[asyncio.AbstractEventLoop] = None,
-        **options: Any,
+        shard_id: Optional[int] = None,
+        shard_count: Optional[int] = None,
+        enable_debug_events: bool = False,
+        localization_provider: Optional[LocalizationProtocol] = None,
+        strict_localization: bool = False,
+        connector: Optional[aiohttp.BaseConnector] = None,
+        proxy: Optional[str] = None,
+        proxy_auth: Optional[aiohttp.BasicAuth] = None,
+        assume_unsync_clock: bool = True,
+        max_messages: Optional[int] = 1000,
+        application_id: Optional[int] = None,
+        heartbeat_timeout: float = 60.0,
+        guild_ready_timeout: float = 2.0,
+        allowed_mentions: Optional[AllowedMentions] = None,
+        activity: Optional[BaseActivity] = None,
+        status: Optional[Union[Status, str]] = None,
+        intents: Optional[Intents] = None,
+        chunk_guilds_at_startup: Optional[bool] = None,
+        member_cache_flags: MemberCacheFlags = None,
```

Comment:
```suggestion
        member_cache_flags: Optional[MemberCacheFlags] = None,
```

Reply:
Fixed in 83ddb4a3785dc78c0d97c8c33f77c5c6dbcff996.